src/darwin/Framework/CHIPTests/MTRPairingTests.m

```diff
@@ -163,6 +163,12 @@ - (void)setUp
     [self setContinueAfterFailure:NO];
 }
 
+- (void)tearDown
+{
+    [sController setDeviceControllerDelegate:(id _Nonnull) nil queue:dispatch_get_main_queue()]; // TODO: do we need a clearDeviceControllerDelegate API?
```

Comment:
Why do we need to clear it?  This isn't something things will need to do normally...  followup to sort this out is ok.

Reply:
It's not strictly necessary here, but leaving stuff like delegates hanging around between tests can be annoying sometimes.